API-1560: Add documentation about getting products with_completenesses (#490)
akeneo/pim-api-docs: content/concepts/products.md

Note that the `quality_scores` field is only available since the 5.0 and when the `with_quality_scores` query parameter is set to `true`.
:::

::: warning

> Note that the `completenesses` field is only available in the PIM Serenity version and when the `with_completenesses` query parameter is set to `true`.

Reviewer: ```suggestion
Note that the `completenesses` field is only available on SaaS platforms, and when the `with_completenesses` query parameter is set to `true`.
```
